docs/INTEGRATION_RECOMMENDATIONS.md

```diff
@@ -18,7 +18,8 @@ Context variables may be applied to individual recommendation profiles similar t
 | product | current product sku | product detail page | used to identify the current product being viewed |
 | cart | array (or function that returns an array) of current cart skus | all | optional method of setting cart contents |
 | options.siteId | global siteId overwrite | all | optional global siteId overwrite |
-| options.categories | category path | all | optional category identifier to restrict recommendations |
+| options.categories | array of category path strings | all | optional category identifier for category trending rec type |
```

Comment:
My comment about this description was a bit misleading, I think we ought to clean this up for customer consumption.

Suggestion:
> optional category identifiers used in category trending recommendation profiles